python/GafferImageUI/CatalogueUI.py

```diff
@@ -465,9 +475,12 @@ def __addClicked( self, *unused ) :
 			self.__images().addChild( GafferImage.Catalogue.Image.load( str( path ) ) )
 			self.getPlug().setValue( len( self.__images() ) - 1 )
 
-	def __metadataIndexToGraphComponentIndex( self, index ) :
+	def __logicalIndexToIndex( self, index ) :
```

Comment:
The meaning of "logical index" here is reversed from my usual understanding. From Qt I'm used to logical indices being where things are in the API (in our case the GraphComponent parent) and visual indices being where they are in the UI.

Reply:
Switched to `UI` in https://github.com/GafferHQ/gaffer/commit/9aed9541e19f8838e8997a7eb8c62f5803530915. I guess (I forget when) the first time I was working on something with that differentiation it was 'logical' and 'physical' index...